### ISSUE TITLE
liquibase generatechangelog not working as expected with enums

### ISSUE THREAD

**Issue by nimishagarwal76** [liquibase/liquibase#2230]
## Environment - Docker - liquibase:4.5.0

**Liquibase Version**: 4.5.0

**Liquibase Integration & Version**: CLI

**Liquibase Extension(s) & Version**: 

**Database Vendor & Version**: mysql 8.0

**Operating System Type & Version**:

## Description

The changelog generated by liquibase generateChangeLog does not work as expected for ENUM. It provides values in reverse order.

## Steps To Reproduce
- Create source db and a database test
- In the database
`CREATE TABLE sizes (
    size ENUM('small', 'large')
);`
- Now run liquibase generateChangeLog

## Actual Behavior
`    <changeSet author="alpha (generated)" id="1637671757006-1">
        <createTable tableName="sizes">
            <column name="size" type="ENUM('large', 'small')"/>
        </createTable>
    </changeSet>`
It creates ENUM large than small

## Expected/Desired Behavior
It directly affects functionlity
https://stackoverflow.com/questions/17603907/order-by-enum-field-in-mysql/17603993

**Comment by molivasdat** [comment]
Hi @nimishagarwal76 Thanks for the alert on this issue.  I think ENUMs are User Defined Types. I could be wrong.  Is the problem that is creates the ENUMs in the wrong order?
If so, we can add it to the list of issues we are processing.
If you have a fix, please create a PR.

**Comment by nimishagarwal76** [comment]
Yes, enums are used defined types. Now if I create enum('A', 'B', 'C'), to have everything consistent, generated sql should be enum('A', 'B', 'C'). But liquibase generates enum(CBA) .
This works fine in stuff like mysqldump.

**Comment by ecarneiro01** [comment]
This is still an issue in 4.15.0.

**Comment by Yu-Qi** [comment]
I also have the same problem in 4.11.0
The order of enum is reversed.
